test/services/RegistrationOrchestrationServiceSpec.scala

```diff
@@ -108,7 +108,8 @@ class RegistrationOrchestrationServiceSpec extends AnyWordSpec
         "the business entity was not challenged to verify" in {
           mockRetrieveNino(testJourneyId)(Future.successful(Some(testNino)))
           mockRetrieveSautr(testJourneyId)(Future.successful(Some(testSautr)))
-          mockRetrieveBusinessVerificationStatus(testJourneyId)(Future.successful(Some(BusinessVerificationUnchallenged)))
+          mockRetrieveBusinessVerificationStatus(testJourneyId)(Future.successful(Some(BusinessVerificationNotEnoughInformationToCallBV)))
```

Comment:
The mock call to retrieve BusinessVerificationStatus is duplicated. I don't think the mocked calls to retrieve the nino and SA utr values are required also.